trino-api/cmd/api/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"trino-api/internal/app"
+	"trino-api/internal/boot"
+
+	log "github.com/sirupsen/logrus"
```

Comment:
switch to uber zap

reason:
the main module is using zap, it will be easier to integrate

unless, logrus provides something which is not possible using zap, which I cant see in this module